modules/service/src/main/scala/ch/epfl/bluebrain/nexus/admin/service/routes/ProjectRoutes.scala

```diff
@@ -54,8 +54,8 @@ final class ProjectRoutes(projects: Projects[Future])(implicit iamClient: IamCli
     new RoutesEncoder[Resource[ProjectReference]]()
   import encoders._
   private def readRoutes(implicit credentials: Option[OAuth2BearerToken]): Route =
-    (segment2(of[ProjectReference]) & pathEndOrSingleSlash) { name =>
-      (get & authorizeOn[HasReadProjects](name.value)) { implicit perms =>
+    (projectReference & pathEndOrSingleSlash) { name =>
+      (get & authorizeOn[HasReadProjects](name.show)) { implicit perms =>
```

Comment:
You could avoid all this `authorizeOn` `.show` by modifying the string signature of [the authorizeOn method](https://github.com/wwajerowicz/nexus-admin/blob/1c3633b193574a7dd7cfa4b2abd43ac5026375ef/modules/service/src/main/scala/ch/epfl/bluebrain/nexus/admin/service/directives/AuthDirectives.scala#L84) to something like
```scala
def apply[F[_, _], P, A: Show](value: A)(
      implicit ev: F[FullAccessControlList, P] =:= FTP,
      rt: RefType[F],
      v: Validate[FullAccessControlList, P],
      client: IamClient[Future],
      cred: Option[OAuth2BearerToken]
  ): Directive1[FTP] = apply(Path(value.show))
```

Reply:
But if you try `authorizeOn` and  pass `Path` object to the apply method e.g. 
```
(pathEndOrSingleSlash & authorizeOn[HasReadProjects](Path.Empty / "*" / "*"))
```
Scala compiler seems to pick up the wrong method: `def apply[F[_, _], P, A: Show](value: A)`
rather than
` def apply[F[_, _], P, A: Show](path: A)`